### PR TITLE
chore: adding generate delta worker to enterprise tests composition

### PR DESCRIPTION
**Pull request by merlin-northern**
Ticket: None
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@lluiscampos this if just fyi

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please

**Comment by merlin-northern** [comment]
@kjaskiewiczz could you run your pipelines from MEN-5806 with this?

File: docker-compose.demo.yml [hunk 1]
```diff
@@ -94,3 +94,8 @@ services:
         command: --automigrate
         environment:
             - CREATE_ARTIFACT_SKIPVERIFY=1
+
+    mender-generate-delta-worker:
```
Comment (kjaskiewiczz) [inline docker-compose.demo.yml:98]: this will prevent starting open source demo, because mender-generate-delta-worker is not an open source component

Comment (lluiscampos) [reply docker-compose.demo.yml:98]: Yes, this cannot go here. Generally speaking we don't have an "enterprise demo" mode. We only have this `--enterprise-testing` flag that we use internally (hence it is not documented).

As this is only needed for tests, it should go into [`docker-compose.testing.enterprise`](https://github.com/mendersoftware/integration/blob/master/docker-compose.testing.enterprise.yml) instead.

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: docker-compose.demo.yml (hunk 1 above)
Comment (merlin-northern) [reply docker-compose.demo.yml]: yes, and also breaks the os test, I also forgot that this is ent only. fixed. sorry for that.

**Review by kjaskiewiczz — APPROVED** [review]

**Comment by merlin-northern** [comment]
some surious failures, unrelated I would say.

**Pull request merged**